### ISSUE TITLE
AR6 inputs for Hector 

### ISSUE THREAD

**Issue by kdorheim** [JGCRI/hectordata#23]
As discussed with @ssmithClimate and @bpbond Hector, users should be easily able to run the historical AR6 & future scenarios.  While not immediately needed for Hector v3, however, this is a high-priority scenario to be able to run. 

AR6 replication scenario would have to be some combination of concentration & emission driven. One idea that @ssmithClimate had would be to run Hector with the concentration constraints & do some sort of inverse calculation to be generated the corresponding emission time series. 

Inputs for the AR6 will come from https://github.com/chrisroadmap/ar6

**Comment by ssmithClimate** [comment]
Sounds good.

One could think of this in terms of replicating the CMIP DECK protocol, whereby for GHGs the run is concentration-driven over the historical period and then emissions driven into the future. (With, as mentioned above, one approach being to do the inverse calculation so the historical emissions-driven run matches historical concentrations.)

One thing we can think about in this context is how to keep this up to date as we move forward. Emissions, and historical concentration, data become updated every year so we eventually might want to eventually think a bit about how to keep hector current in this respect as well.

**Comment by kdorheim** [comment]
Yes having the CMIP DECK protocol readily would be extremely valuable!
